docs/annotation.md

```diff
@@ -0,0 +1,20 @@
+# Annotation
+
+> 🚧 **Note:** This only applies to the callback redesign of Mava.
+
+In MARL, it is usual to have several algorithms go by slightly different names across multiple studies.  So, in order to avoid this confusion in MAVA, we want to clarify some annotation conventions that we used in building our different systems.
```

Comment:
Since merging mappo will not happen before Indaba, I think that this can be removed until we actually follow this. Even in TF, we say MAPPO but it is actually IPPO by default and we change the feed if centralised.

Reply:
> Thanks @RuanJohn and @OmaymaMahjoub . It looks really good rocket . There's just some minor things I'd like to ask about.
> 
> * In debugging, the text doesn't wrap around the image anymore.
> * In debugging ,should there be captions under the text to make it easier to skim through the docs?
> * IPPO vs. MAPPO, "According to previous research, the PPO algorithm in the case of multi-agent can be implemented for both case scenarios" could be worded as "In the multi-agent case, the PPO algorithm can be implemented for both cases".

THanks @siddarthsingh1  for your review:
-About the debugging I fixed the error but the website I share it don't update my changes fast so you need to run the website locally using` mkdoc serve`
- About the IPPO vs. MAPPO done!